launch.go

```diff
@@ -448,6 +454,7 @@ func main() {
 		workspace := c.String("workspace")
 		emitterPath := c.String("emitter")
 		metaSpace := c.String("meta-space")
+		storeURL := c.String("store-uri")
```

Comment:
Store-uri and Store-url coexist. Does it express important difference?

Reply:
Screwdriver api is also coexist uri and url thus I wrote by the way as same as Screwdriver api.

https://github.com/screwdriver-cd/launcher/blob/master/launch.go#L446

Reply:
Understood. Thank you.